### DISCUSSION TITLE
can't install cockpit on vera linux-mint

### DISCUSSION THREAD

**Discussion by 404Nikhil** [cockpit-project/cockpit#19601] [Q&A] [answered]
![image](https://github.com/cockpit-project/cockpit/assets/83774380/1295837f-e68a-4103-a1ac-5ef1c747a06b)

![image](https://github.com/cockpit-project/cockpit/assets/83774380/ffbfff4d-183c-4ac2-aa55-dc8c90028b9d)

It seems that I am following the instructions correctly as outlined in the documentation. Can you please assist me with this? Thank you.

**Comment by martinpitt** [comment, marked as answer]
Indeed it seems that the [pocket name](https://ftp.rz.uni-frankfurt.de/pub/mirrors/linux-mint/packages/dists/vera/) is "backport", not "backports" for Linux mint. Can you try with `-t vera-backport`? That should at least fix the error message.

That said, [Mint vera does not actually have cockpit in backports](https://ftp.rz.uni-frankfurt.de/pub/mirrors/linux-mint/packages/dists/vera/backport/binary-amd64/), apparently they don't follow Ubuntu there.

The same is true for [una](https://ftp.rz.uni-frankfurt.de/pub/mirrors/linux-mint/packages/dists/una/backport/binary-amd64/), which is based on the previous Ubuntu LTS 20.04.

So I think we should just simply drop the Mint instructions.
